src/analyzer/plugins/analyzerpluginsupport.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <QString>
+
+#include "analyzer/analyzer.h"
+#include "analyzer/plugins/analyzerplugin.h"
+
+struct AnalyzerPluginSupportInfo {
```

Comment:
I think this should be a class. We use struct for structs without member functions.